data_fetch.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"os"
+)
+
+var fetcher dataFetch
+
+const (
+	cacherDataModel = `
+	{
+	  "id": "8978e7d4-1a55-4845-8a66-a5259236b104",
```

Comment:
why this id is hard coded here?

Reply:
This is a mock of what would be returned from cacher so that in the tests we don't have to actually make calls to cacher. I'm working on moving these constants into the actual test file so it's not confusing like so.

Reply:
if just for testing, fine for now.